groups/models.py

```diff
@@ -161,6 +167,9 @@ def delete_state(self):
     def is_deleted(self):
         return self.state == self.STATE_DELETED
 
+    def __str__(self):
+        return 'Comment by {}'.format(self.user.get_full_name())
```

Comment:
> `...object has no attribute 'discussion_id'`

Um.... I can't see how that is possible. It has a FK to `Discussion` called `discussion`. In my mind it should follow that it has a field to store the `id` of the related object -- `discussion_id`. Hm. Perhaps it's worth inspecting what data *is* available on the instance?